src/prasad/Findcommonelefrarray.java

```diff
@@ -0,0 +1,25 @@
+package prasad;
+
+public class Findcommonelefrarray {
+	void scanArray(int[]array1 , int[]array2) {
+		if(array1.length!=array2.length)
+			System.out.println("Given two arrays are not of same length, please retry");
+		else
+			System.out.println("Common Elements:- ");
+		for(int index=0;index<array1.length;index++) {
+			if(array1.length != array2.length)
```

Comment:
this condition you already given, then why again here.

Reply:
Hi Prasad,
you need to compare element in two loop, fortunately both array have same common value on same index, but think on diff index,
Pls update accordingly. I know this is not mentioned but this is like counter question on same program.